test/extract-dist.sh

```diff
@@ -23,6 +23,4 @@ npm init -y > /dev/null
 npm install ${CDK8S_DIST}/js/*.tgz
 export PATH=${staging}/node_modules/.bin:$PATH
 
-# restore working directory
```

Comment:
Not sure what's up here. run-against-dist is useful for locally running integ tests against the dist folder.

Reply:
The reason I split it up here was because the `PATH` was not getting carried over from here to the `test-all.sh` script. One way to get the path working was to add it to `$GITHUB_PATH`, but anything added there cannot be used in the same step. So then I split the extract and test into two steps.